habitat-lab/habitat/tasks/rearrange/rearrange_sim.py

```diff
@@ -476,37 +474,29 @@ def _setup_targets(self, ep_info: RearrangeEpisode):
             )
 
     @add_perf_timing_func()
-    def _load_navmesh(self, ep_info: RearrangeEpisode):
-        scene_name = ep_info.scene_id.split("/")[-1].split(".")[0]
-        base_dir = osp.join(*ep_info.scene_id.split("/")[:2])
+    def _recompute_navmesh(self) -> None:
+        """
+        Recompute the navmesh including STATIC objects and using agent parameters.
+        """
 
-        navmesh_path = osp.join(base_dir, "navmeshes", scene_name + ".navmesh")
+        navmesh_settings = NavMeshSettings()
+        navmesh_settings.set_defaults()
 
-        if osp.exists(navmesh_path):
-            self.pathfinder.load_nav_mesh(navmesh_path)
-            logger.info(f"Loaded navmesh from {navmesh_path}")
+        agent_config = None
+        if hasattr(self.habitat_config.agents, "agent_0"):
+            agent_config = self.habitat_config.agents.agent_0
+        elif hasattr(self.habitat_config.agents, "main_agent"):
+            agent_config = self.habitat_config.agents.main_agent
```

Comment:
Is this `main_agent` convention still used?

Reply:
I still see this in several places in the other codebase, so I assume so.